### ISSUE TITLE
Add configurable extension settings

### ISSUE THREAD

**Issue by avanishsubbiah** [avanishsubbiah/material-you-theme#17]
Settings window should allow the user to modify
 - Wallpaper sample resize (width/height)
 - Show notifications on theme update (on/off)
 - Customize color mappings (drop-down menu with material tokens)
 - Customize color opacities (slider for percentages)

**Comment by Radical4ever** [comment]
Yeah something like the theme style!, like the repainter app, from android.
![Screenshot_20220805-082912_Repainter](https://user-images.githubusercontent.com/83252104/183068632-15caf7f9-5ec0-4b42-98d9-28a5019a7a6d.jpg)

**Comment by ardjuh** [comment]
Hi, i made a pull request for the notification setting.
https://github.com/avanishsubbiah/material-you-theme/pull/25#issue-1330373499
Hope it's a good start.

**Comment by avanishsubbiah** [comment]
> Yeah something like the theme style!, like the repainter app, from android. ![Screenshot_20220805-082912_Repainter](https://user-images.githubusercontent.com/83252104/183068632-15caf7f9-5ec0-4b42-98d9-28a5019a7a6d.jpg)

I modified pull request  #25 to add a setting for using vibrant colors, which changes the color mappings to use surface +3,4 primary layers and tertiary colors. This can be see in version 10 of the extension.

I think the next improvement is updating the settings menu to have a selectable list of color mapping choices like the repainter app.

**Comment by avanishsubbiah** [comment]
Version 11 of the extension adds all color styles from the image above, notifications enable/disable, and wallpaper sample configuration (notifications won't trigger on enable() or disable() to avoid security risk mentioned in issue #26)

I'm not exactly sure how to add a formatted text entry and text schema to enable custom css addition from issue #20. So if anyone has ideas on how to do that please post below or submit pr. 

Thanks!

**Comment by Radical4ever** [comment]
Yeah, i saw the update its pretty cool, thank you!
